src/win/fs.c

```diff
@@ -126,7 +126,7 @@ static int uv__usermode_symlink_available(void) {
   }
   pRtlGetVersion(&version_information);
   /* Usermode symlinks are only available on Win10 with Creators Update... */
-  if (version_information.dwMajorVersion < 10 ||
+  if (version_information.dwMajorVersion != 10 ||
```

Comment:
This is still wrong with regard to future proofing. Windows 11 will return zero here.